commit c8d0956bd4094d23e45902aaea76b32d47ea1a13 (refs/changes/69/869/1)
Author: David Pacheco <dap@joyent.com>
Date:   2016-11-09T14:42:41-08:00 (2 years, 11 months ago)
    
    MANTA-3011 marlin worker big theory statement points to the wrong directory for queries

diff --git a/jobsupervisor/lib/worker/worker.js b/jobsupervisor/lib/worker/worker.js
index e1e99f2..723f2f4 100644
--- a/jobsupervisor/lib/worker/worker.js
+++ b/jobsupervisor/lib/worker/worker.js
@@ -169,7 +169,7 @@
  *     for phases 0 ... i - 1, where i = the reducer's phase.
  *
  * The detailed list of queries used for polling can be found in
- * lib/{worker,agent}/queries.js.
+ * agent/lib/agent/queries.js and jobsupervisor/lib/worker/queries.js.
  */
 
 var mod_assert = require('assert');
